src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4729,8 +4729,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_RefPropertyCannotHaveSetAccessor" xml:space="preserve">
     <value>Properties which return by reference cannot have set accessors</value>
   </data>
-  <data name="ERR_CantChangeRefReturnOnOverride" xml:space="preserve">
-    <value>'{0}' must {2}return by reference to match overridden member '{1}'</value>
+  <data name="ERR_CantChangeRefnessOfReturnOnOverride" xml:space="preserve">
```

Comment:
Alright. Will change it back. ESL here :)

---
In reply to: [104769302](https://github.com/dotnet/roslyn/pull/17547#discussion_r104769302) [](ancestors = 104769302)